src/test/resources/net/sf/jabref/logic/importer/fileformat/MsBibImporterTest3.bib

```diff
@@ -1,10 +1,12 @@
+% Encoding: UTF-8
+
 @book{raey,
   author = {5},
-  journal = {Wirtschaftsinformatik},
+  journaltitle = {Wirtschaftsinformatik},
```

Comment:
Just maps the fields from the conversion HashMap from MsBib to the BibLaTex fields, so the test doesn't check for the bibmode.